dubbo-rpc/dubbo-rpc-triple/pom.xml

```diff
@@ -59,6 +59,11 @@
             <version>3.4.16</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>1.21</version>
```

Comment:
1. The `version` needs to be specified in `bom`'s `pom.xml` 
2. May `optional` or `compileOnly` scope will be better to reduce indirect deps of Dubbo? Users can  import it manually.

Reply:
@guohao Thank you for your review